src/osd/osd_types.h

```diff
@@ -6071,7 +6071,7 @@ std::ostream& operator<<(std::ostream& out, const PushOp &op);
 /*
  * summarize pg contents for purposes of a scrub
  *
- * If members are added to ScrubMao, make sure to modify swap().
+ * If members are added to ScrubMap, make sure to modify swap().
```

Comment:
Oops, this landed outside the commit that introduced the comment (the first one).

Reply:
Fixed. Thanks